src/player.py

```diff
@@ -1,2 +1,46 @@
 # Write a class to hold player information, e.g. what room they are in
 # currently.
+
+
+class Player:
+    def __init__(self, name, starting_room):
+        self.name = name
+        self.current_room = starting_room
+        self.hp = 15
+        self.mp = 15
+        self.items = []
+        self.spells = [
```

Comment:
this looks like an interesting idea for the game that ive not seen anyone implement yet.